#
# Ubuntu: https://bugs.launchpad.net/bugs/429099
# Patch: http://www.abisource.com/viewvc/abiword-plugins/branches/ABI-2-6-0-STABLE/tools/abicommand/xp/AbiCommand.cpp?r1=22067&r2=25893
# Description: abiword FTBFS with gcc 4.4, patch can be dropped when
#              upgrading to the next upstream version (> 2.6.8)
#
--- abiword-2.6.8/../abiword-plugins-2.6.8/tools/abicommand/xp/AbiCommand.cpp	2007-10-11 22:05:55.000000000 +0200
+++ abiword-2.6.8/../abiword-plugins-2.6.8/tools/abicommand/xp/AbiCommand.cpp	2009-09-14 00:03:31.000000000 +0200
@@ -551,7 +551,7 @@
 				UT_UTF8String *pCom1 =
 					const_cast < UT_UTF8String * >(static_cast < const UT_UTF8String * >(pToks->getNthItem (1)));
 				printf(" Filename %s \n",pCom1->utf8_str());
-				char *suffix = rindex (pCom1->utf8_str (), '.');
+				const char *suffix = rindex (pCom1->utf8_str (), '.');
 
 				if (suffix != NULL)
 				{
